### ISSUE TITLE
Anyone tried AX750? I'm close...

### ISSUE THREAD

**Issue by modemmike** [TalalMash/Unofficial-Speedify-Installer-For-OpenWrt#12]
How I installed it (needed to use the sd card for space)

opkg update && \
opkg install block-mount kmod-fs-ext4 kmod-usb-storage e2fsprogs && \

umount /dev/sda1
mkfs.ext4 /dev/sda1
mkdir -p /mnt/sda1
mount /dev/sda1 /mnt/sda1
cp -a -f /overlay/. /mnt/sda1
cat << EOF > /etc/config/fstab
config 'global'
    option anon_swap '0'
    option anon_mount '0'
    option auto_swap '1'
    option auto_mount '1'
    option delay_root '5'
    option check_fs '0'

config 'mount'
    option target '/overlay'
    option device '/dev/sda1'
    option fstype 'ext4'
    option options 'rw,sync'
    option enabled '1'
    option enabled_fsck '0'
EOF
/etc/init.d/fstab enable
reboot

opkg install ar xz xz-utils tar luci-compat libcap wget-ssl

Do the installation steps outlined in the [video](Do the installation steps outlined in the video) (uploaded the ipk etc).

From the log:

Log Reset & Speedify Update/Install
Repository URL:https://apt.connectify.me
Repository Ubuntu packages URL:https://apt.connectify.me/dists/speedify/main/binary-armhf/Packages
Latest Version:14.8.5-11916
Speedify package URL:https://apt.connectify.me/pool/main/s/speedify/speedify_14.8.5-11916_armhf.deb
Speedify UI package URL:https://apt.connectify.me/pool/main/s/speedifyui/speedifyui_14.8.5-11916_armhf.deb
Installing GNU C Library
Removing cache if it exists at /tmp/spddw
Downloading Speedify...
Downloading Speedify UI...
Extracting Speedify...
Extracting Speedify UI...
Deleting installation files...
Updating OpenWrt configration and starting Speedify...
Speedify is now installed, user interface is at http://<routerip>/spdui/?wsPort=9331

Unfortunately, all I get for the SpeedifyUI is

![image](https://github.com/TalalMash/Unofficial-Speedify-Installer-For-OpenWrt/assets/5247152/dbc4d8b5-2e78-4cb3-94c0-b02553f48fa6)

I like this router because it's small, low power and can be found used for less than $50. My plan was to combine the onboard lte, add a USB modem and then also mix in wifi when available.

**Comment by TalalMash** [comment]
GL.iNet Slate AR750 or the Spitz? The non AX versions use MIPS processor which isn't supported by Speedify.

**Comment by modemmike** [comment]
Yep, Mips... that solves that. Thank you. Darn! Mine is MIPS... [GL iNet x750](https://openwrt.org/toh/gl.inet/gl-x750) I have the original version, not the version 2. 

Got any suggestions for small package and low power?

**Comment by TalalMash** [comment]
The Wi-Fi AX versions are all ARM, the Mediatek GL.iNet series may use less power than the qualcomm and is cheaper.
I've had a working report from a Spitz AX user.

**Comment by modemmike** [comment]
The $500 Spitz? Ya, thats a bit rich for me.
GL-E750V2 is a nice package for my use case and is ARM, worth a shot you think?

**Comment by TalalMash** [comment]
Yikes yeah that's a lot, Banana Pi R3 Mini has a PCIe slot for a modem that may be cheaper.
The Mudi(v2) is also MIPS.

**Comment by modemmike** [comment]
The Pi Zero is interesting. Thank you for pointing me in the right direction.

**Comment by TalalMash** [comment]
Speedify mentioned that they are going to release MIPS support soon, so the old Slate and tons of other routers could probably work well at low/streaming speeds.

**Comment by modemmike** [comment]
IDK if the processor in the AX750 can handle it but that would be cool.

**Comment by modemmike** [comment]
Did they mention a potential ETA? This AX3000 is power hungry, relatively speaking. I'm tweaking the settings and getting that lower but it's also twice the size.
